lib/structures/TextChannel.js

```diff
@@ -124,29 +124,48 @@ class TextChannel extends GuildChannel {
+     */
+    createThread(options) {
+        return this.client.createThread.call(this.client, this.id, options);
+    }
+
     /**
      * Create a thread with an existing message
      * @arg {String} messageID The ID of the message to create the thread from
      * @arg {Object} options The thread options
-     * @arg {Number} options.autoArchiveDuration Duration in minutes to automatically archive the thread after recent activity, either 60, 1440, 4320 or 10080
+     * @arg {Number} [options.autoArchiveDuration] Duration in minutes to automatically archive the thread after recent activity, either 60, 1440, 4320 or 10080
      * @arg {String} options.name The thread channel name
+     * @arg {Number} [options.rateLimitPerUser] The time in seconds a user has to wait before sending another message (0-21600) (does not affect bots or users with manageMessages/manageChannel permissions)
+     * @arg {String} [options.reason] The reason to be displayed in audit logs
      * @returns {Promise<NewsThreadChannel | PublicThreadChannel>}
      */
     createThreadWithMessage(messageID, options) {
         return this.client.createThreadWithMessage.call(this.client, this.id, messageID, options);
     }
 
     /**
-     * Create a thread without an existing message
+     * [Deprecated] Create a thread without an existing message. Use `createThread` instead
      * @arg {Object} options The thread options
-     * @arg {Number} options.autoArchiveDuration Duration in minutes to automatically archive the thread after recent activity, either 60, 1440, 4320 or 10080
+     * @arg {Number} [options.autoArchiveDuration] Duration in minutes to automatically archive the thread after recent activity, either 60, 1440, 4320 or 10080
      * @arg {Boolean} [options.invitable] Whether non-moderators can add other non-moderators to the thread (private threads only)
      * @arg {String} options.name The thread channel name
-     * @arg {Number} [options.type] The channel type of the thread to create. It is recommended to explicitly set this property as this will be a required property in API v10
-     * @returns {Promise<PrivateThreadChannel>}
+     * @arg {Number} [options.rateLimitPerUser] The time in seconds a user has to wait before sending another message (0-21600) (does not affect bots or users with manageMessages/manageChannel permissions)
+     * @arg {String} [options.reason] The reason to be displayed in audit logs
+     * @arg {Number} [options.type] The channel type of the thread to create. This property defaults to private thread (12), but it is recommended to explicitly set this property as this will be a required property in API v10
+     * @returns {Promise<NewsThreadChannel | PublicThreadChannel | PrivateThreadChannel>}
      */
     createThreadWithoutMessage(options) {
-        return this.client.createThreadWithoutMessage.call(this.client, this.id, options);
+        return this.client.createThread.call(this.client, this.id, options);
```

Comment:
Preserve existing functionality
```suggestion
        return this.client.createThreadWithoutMessage.call(this.client, this.id, options);
```

Reply:
It will still function the same regardless 👍🏻